layouts/generic-py-base.html

```diff
@@ -31,10 +31,8 @@
         </div>
       </div>
       {% block global_subheader_container %}
-        <div class="global-subheader">
-          <div class="container">
-            {% block global_subheader %}{% endblock %}
-          </div>
+        <div class="global-subheader container">
```

Comment:
I think that generated doc might be a little outdated. Also, it's just an example of usage. It doesn't have to use the base template, but it helps to be consistent, yeah.